### PR TITLE
Add redirect to new website

### PR DESCRIPTION
**Pull request by maltejur**
This commit adds a redirect from [TheAlgorithms.github.io](https://thealgorithms.github.io/) to [the-algorithms.com](https://the-algorithms.com/). The old page would still be reachable under TheAlgorithms.github.io/index-old.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,186 +1,3 @@
...
-    integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous">
-  </script>
-
-
-</body>
-
-</html>
+<script>
+    window.location = "https://the-algorithms.com/";
+</script>
```
Comment (Panquesito7) [inline index.html]: Missing endline.
```suggestion
</script>

```

**Review by Panquesito7 — COMMENTED** [review]
Does it redirect when you go to the [ideas](https://thealgorithms.github.io/ideas) page?

**Comment by maltejur** [comment]
No it doesn't. It just redirects when you visit the root of the page

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 👍 🎉

**Comment by cclauss** [comment]
At the bottom of https://the-algorithms.com, it says that https://vercel.com is sponsoring it.
Are they paying money to TheAlgorithms for use of our content and logo? @AnupKumarPanwar

**Comment by maltejur** [comment]
No, they just provide free hosting

**Comment by cclauss** [comment]
This does not match with the commitment made by @kvedala that he would cover these costs.  https://github.com/TheAlgorithms/TheAlgorithms.github.io/issues/15#issue-653358814 Giving away our rights to a for-profit company is not what we agreed to.

**Comment by maltejur** [comment]
No rights have been given away. Vercel just provides free hosting on their platform in exchange for the link in the footer. Maybe we should change to wording to "Hosting provided by vercel". But if you really don't like vercel, we can also switch to github pages.

**Comment by maltejur** [comment]
@kvedala also just offered to buy a domain, which @AnupKumarPanwar now did

**Comment by kvedala** [comment]
> No, they just provide free hosting

Isnt "hosting" also done by github pages?

**Comment by Panquesito7** [comment]
> Isn't "hosting" also done by GitHub pages?

_Offtopic:_ We already use [Vercel](https://vercel.com) for hosting and deploying. If you want the website to use GitHub Pages, you may want to contact @AnupKumarPanwar, then.

**Comment by maltejur** [comment]
> Isnt "hosting" also done by github pages?

Yes, but we would have to write a workflow that builds and deploys the site to github pages (that should not be too hard) and there wouldn't be preview deployments on pull requests (that's a Vercel feature which I think is pretty nice).

**Pull request merged**